### PR TITLE
Add tasks 1-10

### PR DESCRIPTION
**Pull request by usersadvocate**

### CODE REVIEW

File: tasks/10_remove-whitespaces/remove-whitespaces.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function trimWhiteSpaces(str) {
+    return str.replace(/\s/g, '');
+}
+
+export {trimWhiteSpaces};
```
Comment (savchukm) [inline tasks/10_remove-whitespaces/remove-whitespaces.js]: Please add an empty line at the end of each file. It is good practice

File: tasks/1_golf-score/golf-score.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    } else if (strokes === par - 1) {
+      return GOLF_SCORES.BIRDIE;
+    } else if (strokes === par) {
+      return GOLF_SCORES.PAR;
+    } else if (strokes === par + 1) {
+      return GOLF_SCORES.BOGEY;
+    } else if (strokes === par + 2) {
+      return GOLF_SCORES.DOUBLE_BOGEY;
+    } else {
+      return GOLF_SCORES.GO_HOME;
```
Comment (savchukm) [inline tasks/1_golf-score/golf-score.js:26]: Switch case construction will be useful here

File: tasks/3_is-boolean/is-boolean.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+function isBoolean (bool) {
+    if (bool === true || bool === false) {
+        return true
```
Comment (savchukm) [inline tasks/3_is-boolean/is-boolean.js]: Consider using `typeof`

File: tasks/3_is-boolean/is-boolean.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+function isBoolean (bool) {
```
Comment (savchukm) [inline tasks/3_is-boolean/is-boolean.js]: No need for space before opening parentheses

File: tasks/5_profile-lookup/profile-lookup.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
+var contacts = [
```
Comment (savchukm) [inline tasks/5_profile-lookup/profile-lookup.js]: ES6 syntax is preferred

File: tasks/5_profile-lookup/profile-lookup.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+        "number": "unknown",
+        "likes": ["Javascript", "Gaming", "Foxes"]
+    }
+];
+
+function lookupProfile(firstName, prop){
+    for(var i in contacts){
+      if(contacts[i].firstName === firstName) {
+        if (contacts[i][prop])
+          return contacts[i][prop];
```
Comment (savchukm) [inline tasks/5_profile-lookup/profile-lookup.js]: `if else` blocks should have `{}` like:
```
if (condition) {
    doSomeStuff()
} else {
    doAnotherStuff()
}
```

**Review by savchukm — CHANGES_REQUESTED** [review]

File: tasks/2_convert-to-f/convert-to-f.js [hunk 7]
```diff
@@ -0,0 +1,6 @@
+function convertToF(celsius) {
+  let fahrenheit = celsius * 9 / 5 + 32;
```
Comment (alexkond) [inline tasks/2_convert-to-f/convert-to-f.js]: It is better to use `const` if you don't modify a variable in your code.

File: tasks/9_find-numbers/find-numbers.js [hunk 8]
```diff
@@ -0,0 +1,5 @@
+function getNumbers(str) {
+  let regex = /\d+/g;
```
Comment (alexkond) [inline tasks/9_find-numbers/find-numbers.js]: It is better to use `const` if you don't modify a variable in your code.

**Review by alexkond — DISMISSED** [review]

File: tasks/3_is-boolean/is-boolean.js [hunk 9]
```diff
@@ -0,0 +1,9 @@
+function isBoolean(bool) {
+  if (typeof(bool) === 'boolean') {
```
Comment (savchukm) [inline tasks/3_is-boolean/is-boolean.js]: `typeof` is an operator. Parentheses is not required here. Moreover if it is simple condition - probably you can just return it

File: tasks/7_random-fraction/random-fraction.js [hunk 10]
```diff
@@ -0,0 +1,4 @@
+function randomFraction() {
+  return Math.random();
```
Comment (savchukm) [inline tasks/7_random-fraction/random-fraction.js]: Example of `Math.random()` call:
0.8095752234458309
We need to get random of 0.1, 0.2,...,0.9

**Review by savchukm — CHANGES_REQUESTED** [review]

**Review by alexkond — APPROVED** [review]

**Review by savchukm — APPROVED** [review]

File: tasks/5_profile-lookup/profile-lookup.js [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+    'lastName': 'Vos',
+    'number': 'unknown',
+    'likes': ['Javascript', 'Gaming', 'Foxes'],
+  },
+];
+
+function lookupProfile(firstName, prop) {
+  for (let i in contacts) {
+    if (contacts[i].firstName === firstName) {
+      if (contacts[i][prop]) {
```
Comment (KalachevDev) [inline tasks/5_profile-lookup/profile-lookup.js:31]: You also could use
`return contacts[i][prop] || 'No such property'`;

File: tasks/7_random-fraction/random-fraction.js [hunk 12]
```diff
@@ -0,0 +1,4 @@
+function randomFraction() {
+  return +(Math.random().toFixed(1));
```
Comment (KalachevDev) [inline tasks/7_random-fraction/random-fraction.js:2]: Please consider using cast by `Number()` just to make your code more readable

**Review by KalachevDev — APPROVED** [review]

**Pull request merged**